runelite-api/src/main/java/net/runelite/api/AnimationID.java

```diff
@@ -168,4 +168,6 @@
 	public static final int ROCKSLUG_DEATH = 1568;
 	public static final int ZYGOMITE_DEATH = 3327;
 	public static final int IMP_DEATH = 172;
+	//Incense burners
```

Comment:
Space after // 

Also you can probably just stick this next to most of the uncategorized animations, its not like there will be more than 1 incense burner animation ever